flink-table-store-core/src/main/java/org/apache/flink/table/store/file/utils/FileUtils.java

```diff
@@ -182,4 +180,20 @@ public static Stream<Long> listVersionedFiles(Path dir, String prefix) throws IO
                 .filter(name -> name.startsWith(prefix))
                 .map(name -> Long.parseLong(name.substring(prefix.length())));
     }
+
+    public static BulkFormat.Reader<RowData> createFormatReader(
+            BulkFormat<RowData, FileSourceSplit> format, Path file) throws IOException {
+        if (!file.getFileSystem().exists(file)) {
+            throw new FileNotFoundException(
+                    String.format(
+                            "File '%s' not found, Possible causes: "
+                                    + "1.consumption is too slow, you can improve the performance of consumption. "
+                                    + "2.snapshot expires too fast, you can configure a larger snapshot.time-retained.",
```

Comment:
```suggestion
                                    + "2.For the situation that snapshot expires too fast, you can configure 'snapshot.time-retained' option with a larger value.",
```

Reply:
For `For the situation that`, I think the reason is a combination, there is no need to emphasize `the situation`.